### DISCUSSION TITLE
v0.8.5

### DISCUSSION THREAD

**Discussion by justinxzhao** [ludwig-ai/ludwig#3708] [General]
## What's Changed
* Add function to free GPU memory by @Infernaught in https://github.com/ludwig-ai/ludwig/pull/3643
* ❗ Enable LLM fine-tuning tests when no quantization is specified by @arnavgarg1 in https://github.com/ludwig-ai/ludwig/pull/3626
* Add check to ensure selected backend works with quantization for LLMs  by @arnavgarg1 in https://github.com/ludwig-ai/ludwig/pull/3646
* [CI] Use a torch-nightly-compatible version of torchaudio by @justinxzhao in https://github.com/ludwig-ai/ludwig/pull/3644
* Set do_sample default to True by @Infernaught in https://github.com/ludwig-ai/ludwig/pull/3641
* FIX: Failure in audio feature related test by @jimthompson5802 in https://github.com/ludwig-ai/ludwig/pull/3651
* Remove unnecessary peft config updating by @Infernaught in https://github.com/ludwig-ai/ludwig/pull/3642
* FIX: docker build error for ludwig-gpu by @jimthompson5802 in https://github.com/ludwig-ai/ludwig/pull/3658
* Exclude getdaft on Windows by @carlogrisetti in https://github.com/ludwig-ai/ludwig/pull/3629
* Add daft back for windows since the wheels are now officially published by @arnavgarg1 in https://github.com/ludwig-ai/ludwig/pull/3663
* fix: The final batch of an epoch is skipped when batch size is 1 by @jeffkinnison in https://github.com/ludwig-ai/ludwig/pull/3653
* Place metric functions for BLEU and Rogue on correct devices when using multiple GPUs by @arnavgarg1 in https://github.com/ludwig-ai/ludwig/pull/3671
* Remove duplicate metrics by @Infernaught in https://github.com/ludwig-ai/ludwig/pull/3670
* Increment epochs based on last_batch() instead of at the end of the train loop. by @justinxzhao in https://github.com/ludwig-ai/ludwig/pull/3668
* [FEATURE] Support Merging LoRA Weights Into Base Model (Issue-3603) by @alexsherstinsky in https://github.com/ludwig-ai/ludwig/pull/3649
* [FEATURE] Include Mistral-7B model in list of supported base models by @alexsherstinsky in https://github.com/ludwig-ai/ludwig/pull/3674
* [MAINTENANCE] Partially reconcile type hints, fix some warnings, and fix comments in parts of the codebase. by @alexsherstinsky in https://github.com/ludwig-ai/ludwig/pull/3673
* Improve error message for when an LLM base model can't be loaded. by @justinxzhao in https://github.com/ludwig-ai/ludwig/pull/3675
* Fix eos_token and pad_token issue by @Infernaught in https://github.com/ludwig-ai/ludwig/pull/3667
* FIX: error with nightly CI tests for test_resize_image by @jimthompson5802 in https://github.com/ludwig-ai/ludwig/pull/3678
* [BUGFIX] Remove spurious test directory at the end of the test_llm.py::test_local_path_loading test run by @alexsherstinsky in https://github.com/ludwig-ai/ludwig/pull/3680
* Add per-device logging to tensorboard by @Infernaught in https://github.com/ludwig-ai/ludwig/pull/3677
* Fix dynamic generation config load during `model.predict` by @geoffreyangus in https://github.com/ludwig-ai/ludwig/pull/3666
* [CI] Ensure that mlflow callback cleans up background-saving threads on trainer teardown. by @justinxzhao in https://github.com/ludwig-ai/ludwig/pull/3683
* fix: temporarily remove config validation check for backend by @geoffreyangus in https://github.com/ludwig-ai/ludwig/pull/3688
* fix: Failing test for backend with quantization by @arnavgarg1 in https://github.com/ludwig-ai/ludwig/pull/3689
* [BUGFIX] Ensure that full base models and not only adapter weights get saved when merge_and_unload is set by @alexsherstinsky in https://github.com/ludwig-ai/ludwig/pull/3679
* Add Ludwig Star History to README by @arnavgarg1 in https://github.com/ludwig-ai/ludwig/pull/3696
* Use sphinx for all docstrings in api.py by @justinxzhao in https://github.com/ludwig-ai/ludwig/pull/3693
* Fix binary variables being visualized as 0 and 1 by @Infernaught in https://github.com/ludwig-ai/ludwig/pull/3691
* [MAINTENANCE] Fix the linting warnings in two backend component classes. by @alexsherstinsky in https://github.com/ludwig-ai/ludwig/pull/3698
* [BUGFIX] Pin deepspeed<0.11, skip Horovod tests by @alexsherstinsky in https://github.com/ludwig-ai/ludwig/pull/3700
* Unpin deepspeed following fix in v0.11.1 by @tgaddair in https://github.com/ludwig-ai/ludwig/pull/3706
* Move on_epoch_end and epoch increment to after run_evaluation loop. by @justinxzhao in https://github.com/ludwig-ai/ludwig/pull/3690
* Remove model_load_path from experiment by @Infernaught in https://github.com/ludwig-ai/ludwig/pull/3707
* [FEATURE] Allow typehints without the quotes. by @alexsherstinsky in https://github.com/ludwig-ai/ludwig/pull/3699

## New Contributors
* @alexsherstinsky made their first contribution in https://github.com/ludwig-ai/ludwig/pull/3649

**Full Changelog**: https://github.com/ludwig-ai/ludwig/compare/v0.8.4...v0.8.5

<hr /><em>This discussion was created from the release <a href='https://github.com/ludwig-ai/ludwig/releases/tag/v0.8.5'>v0.8.5</a>.</em>
